### ISSUE TITLE
Issue while using it with d3 and other svg element

### ISSUE THREAD

**Issue by mantri-govind** [sandywalker/webui-popover#100]
Hi,
  I'm using popover on mouse click on some (d3 charts) svg element. On fresh screen it is working properly once I scroll my window to other part of svg its shape get distorted. 
 Before Scroll:
![image](https://cloud.githubusercontent.com/assets/5424160/9083541/b7462096-3b88-11e5-8576-61ae535f6700.png)

After scroll it is like :
 
![image](https://cloud.githubusercontent.com/assets/5424160/9083559/e3e10260-3b88-11e5-972a-eace5c1f2f28.png)

My code is as follows to initialize the popover
 
                        $(this).webuiPopover({   
                        type:'async',
                        closeable:true,
                        contrains:"horizontal",
                        url:'https://api.github.com/',
                        dismissible:true, 
                        content:function(){
                        var html ='<ul class="popup-main">';
                             html+='<li class="popup"> <a class="ajax btn-primary" href="stationTimeTable.html" title="Station Time Table" onClick=getTimeTable("'+ d+'")><i class="fa fa-calendar"></i>  Time Table</a></li>';
                             html+='<li class="popup"> <a class="ajax btn-info"> <i class="fa fa-sign-in"></i>  Open Schematic</a></li>';
                             html+='<li class="popup "> <a class="iframe btn-danger" href="http://wikipedia.com" onClick="imposeSpeed()"><i class="fa fa-hand-paper-o"></i>  Speed Restriction</a></li>';
                             html+='</ul>';
                            return html;
                        }, 
                        trigger:'manual'
                    });  
                       $(this).webuiPopover('show');  
                  });

**Comment by sandywalker** [comment]
It may be caused by some styles conflict, or some class missing when page scroll, I'm not really sure yet.
Could you show me some debug info by chrome dev tool?

**Comment by mantri-govind** [comment]
I got the issue it is cause of width which I had to pass while initializing the pop up according to the amount of content as it is not taking width as auto. Since I made changes in CSS and gave it in some % it was causing that issue on scroll of screen. It would have been better if it could resize according to the content even if some of the element's position is Fixed/absolute to screen.
Thanks.
